manual/python/index.md

```diff
@@ -66,9 +66,95 @@ caption="Example of graphics generated with PyROOT."
 
 ## Loading user libraries & jitting
 
-*Content is coming soon!*
+PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
+
+### Jitting strings with code
+
+ROOT comes with a C++ interpreter to which we can request to process a given piece of C++ code. Sometimes, if such code is short (e.g. the definition of a small function or class) or for rapid exploration/debugging, it is enough the place the C++ code in a Python string and give it to the interpreter. The code will be just-in-time compiled (jitted) and immediately available for invocation, as shown in the example below.
```

Comment:
Put a link to cling and/or the cling docs and/or the respective part in the manual!

Reply:
Done!